src/main/webapp/js/instructorFeedbackResults.js

```diff
@@ -209,15 +209,43 @@ function toggleCollapse(e, pans) {
     var expand = 'Expand';
     var collapse = 'Collapse';
     var panels = pans || $('div.panel-collapse');
-
-    if ($(e).html().trim().startsWith(expand)) {
+    var shouldExpandPanels = $(e).html().trim().startsWith(expand);
+
+    if (shouldExpandPanels) {
+        // toggleCollapse binds with clicking on both collapse panels button(button for all panels)
+        // and collapse panels button section (button for one panel).
+        var isCollapsePanelsButtonClicked = $(e).is($('#collapse-panels-button'));
+
+        // Expand each panel if not yet expanded.
+        // When a panel has class ajax_auto or auto-response-auto, the panel data has not been loaded yet.
+        // We will load the data  by ajax and ajax_auto or auto-response-auto class will be removed.
```

Comment:
And can you add a comma after ajax, so that it's 
`We will load the data by ajax, and ajax_auto or ajax-response-auto class will be removed.` Makes what is being loaded and removed clearer.